File: 42584. 주식가격/주식가격.java
import java.util.Arrays;
class Solution {
    public int[] solution(int[] prices) {
        int length = prices.length;
        int[] answer = new int[prices.length];

        for(int i = 0; i < length; i++){
            int j;
            for(j = i + 1 ; j < length; j++){
                if(prices[i] > prices[j]){
                    answer[i] = j - i;
                    break;
                }
            }
            if(j == length) answer[i] = j - i - 1;
        }
        return answer;
    }
}

File: 2주차/42584. 주식가격/주식가격.py
from collections import deque

def solution(prices):
    answer=[0]*len(prices)

    for i in range(0, len(prices)):
        for j in range(i, len(prices)):
            if prices[i]>prices[j]:
                answer[i]=j-i
                break

        if j==len(prices)-1:
            answer[i]=j-i


    return answer